base/src/com/google/idea/blaze/base/sync/aspects/BlazeBuildOutputs.java

```diff
@@ -43,10 +42,7 @@
  */
 public interface BlazeBuildOutputs {
 
-  @Deprecated
-  ImmutableSet<OutputArtifact> getTargetArtifacts(String label);
```

Comment:
I decided to drop this, since it would require even more changes to the AOSP stuff and would introduce more merge conflicts in future. I am mostly sure that the default output group is correct for the runners.